gabc/details.html

```diff
@@ -89,7 +89,7 @@ <h3 id="liquescences">Liquescentiae and <em>initio debilis</em> figures</h3>
 <p>The notation of an <em>initio debilis</em> is simple: you simply add a hyphen (<strong><code>&nbsp;-&nbsp;</code></strong>) before the note to be affected.</p>
 
 <div class="legende">
-<img src="/illus/initiodebilis.png" title="" alt="" />
+<img src="../illus/initiodebilis.png" title="" alt="" />
 <br /><em>Torculus</em> and <em>torculus initio debilis</em>&nbsp;: <code>fgf -fgf</code>
 </div>
 
```

Comment:
I think `by characters` is more accurately expressed `by a character`.  Also, in this case I think `end of the neumes` should either be `end of the notes` or `end of the neume` (singular).